scripts/bondora/study_configs/2dclustering_W_example_study.yml

```diff
@@ -0,0 +1,143 @@
+study_name: "2dclustering_W_example_study" # Name of this study
+studies_dir: "/storage/epp2/phsajw/bondora/study_results" # Where to make the directory to put study outputs (DB and copy of config file)
```

Comment:
Could the initial part of these paths be supplied by an envinment variable (or a single referencable yaml variable at the start?). The large number of absolute paths that will need changing here feels error prone.